sqlglot/expressions.py

```diff
@@ -43,14 +43,14 @@ class Expression(metaclass=_Expression):
 
     key = "Expression"
     arg_types = {"this": True}
-    __slots__ = ("args", "parent", "arg_key", "type", "comment")
+    __slots__ = ("args", "parent", "arg_key", "type", "comments")
 
     def __init__(self, **args):
         self.args = args
         self.parent = None
         self.arg_key = None
         self.type = None
-        self.comment = None
+        self.comments = []
```

Comment:
i hope this doesn't impact performance

Reply:
I'll revisit this and try to get rid of the empty list in favor of None. For now I just wanted to change it so that it works like before with the new representation.